kratos/python/add_mesh_to_python.cpp

```diff
@@ -602,6 +611,11 @@ void  AddMeshToPython(pybind11::module& m)
     .def("SetValuesOnIntegrationPoints", SetValuesOnIntegrationPointsConstitutiveLaw)
     .def("SetValuesOnIntegrationPoints", SetValuesOnIntegrationPointsDouble<Element>)
     .def("SetValuesOnIntegrationPoints", SetValuesOnIntegrationPointsArray1d<Element>)
+    // SetValueOnIntegrationPoints
+    .def("SetValueOnIntegrationPoints", SetValuesOnIntegrationPointsVector<Element>)
+    .def("SetValueOnIntegrationPoints", SetValuesOnIntegrationPointsConstitutiveLaw)
+    .def("SetValueOnIntegrationPoints", SetValuesOnIntegrationPointsDouble<Element>)
+    .def("SetValueOnIntegrationPoints", SetValuesOnIntegrationPointsArray1d<Element>)
```

Comment:
```suggestion
    .def("SetValuesOnIntegrationPoints", SetValuesOnIntegrationPointsArray1d<Element>)
```

this is proposed in #2194

Reply:
This is exactly what I don't want as it different as in the c++ code...

Reply:
my point is to have one ... so if in the C++ code is without s, i would leave everything without s

Reply:
ah ok, I didn't get this
So currently in C++ it is `SetValueOnIntegrationPoints` and in python it is `SetValuesOnIntegrationPoints`?